pkg/scheduler/schedulerpb/scheduler.proto

```diff
@@ -83,7 +93,10 @@ message FrontendToScheduler {
 
   // Following are used by ENQUEUE only.
   string userID = 4;
-  httpgrpc.HTTPRequest httpRequest = 5;
+  oneof payload {
```

Comment:
Note to reviewers: [same comment as above](https://github.com/grafana/mimir/pull/12302/files#r2255977246) - I've confirmed this is backwards and forwards compatible.